forage-android/src/test/java/com/joinforage/forage/android/network/data/LaunchDarklyTest.kt

```diff
@@ -1,84 +1,77 @@
-package com.joinforage.forage.android.network
+package com.joinforage.forage.android.network.data
 
 import android.app.Application
 import androidx.test.platform.app.InstrumentationRegistry
+import com.joinforage.forage.android.ALWAYS_BT_PERCENT
+import com.joinforage.forage.android.ALWAYS_VGS_PERCENT
 import com.joinforage.forage.android.LDFlags
 import com.joinforage.forage.android.LDManager
 import com.joinforage.forage.android.VaultType
-import com.joinforage.forage.android.core.telemetry.Log
+import com.joinforage.forage.android.computeVaultType
 import com.launchdarkly.sdk.LDValue
+import com.launchdarkly.sdk.android.LDConfig
 import com.launchdarkly.sdk.android.integrations.TestData
-import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.test.runTest
 import org.assertj.core.api.Assertions.assertThat
-import org.junit.After
-import org.junit.BeforeClass
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
 
-@OptIn(ExperimentalCoroutinesApi::class)
+val LD_MOBILE_KEY = "some key"
+
 @RunWith(RobolectricTestRunner::class)
 class LaunchDarklyTest() {
-    private val alwaysBT = 100.0
-    private val alwaysVGS = 0.0
 
+    // NOTE: It is essential that the TestData source live in a
```

Comment:
Just a note on this, I had issues with these tests running with shared state when I was originally creating them. I don't have a full understanding of Kotlin, but I believe Kotlin companion objects are static singletons "whose properties and functions are tied to a class but not to the instance of that class". I guess that's why this causes the tests to function correctly? The more ya know 🤷‍♂️

Reply:
That would make sense but also i have NO IDEA lmao. Figure I at least document the bizarre behavior in a comment for anybody else modifying the tests